resource/system-test/projects.test.js

```diff
@@ -1,30 +1,30 @@
-// Copyright 2016, Google, Inc.
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/**
+ * Copyright 2016, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
 'use strict';
 
-var program = require('../projects');
+const path = require(`path`);
+const run = require(`../../utils`).run;
 
-describe('resource:projects', function () {
-  describe('list', function () {
-    it('should list projects', function (done) {
-      program.listProjects(function (err, projects) {
-        assert.ifError(err);
-        assert(Array.isArray(projects));
-        assert(projects.length > 0);
-        assert(console.log.calledWith('Found %d project(s)!', projects.length));
-        setTimeout(done, 2000);
-      });
-    });
+const cwd = path.join(__dirname, `..`);
+const cmd = `node projects.js`;
+
+describe(`resource:projects`, () => {
```

Comment:
I like this for now - but at some point we need to talk to the team about what language we should write the global system tests in.

(I'm in favor of Node or Python, but I may be biased. :stuck_out_tongue_winking_eye:)